### PR TITLE
WIP: basic CRUD for pc templates

### PR DESCRIPTION
**Pull request by mmacata**

### CODE REVIEW

File: actinia_gdi/api/actinia_modules.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        return make_response(jsonify(actinia_module), 201)
+
+
+class ActiniaModuleId(ResourceBase):
+    """ Manage actinia modules (process chain templates)
+    """
+
+    # @swagger.doc(TODO)
+    def get(self, module_id):
+        """Describe an actinia module (process chain templat).
```
Comment (neteler) [inline actinia_gdi/api/actinia_modules.py]:
```suggestion
        """Describe an actinia module (process chain template).
```

File: actinia_gdi/api/actinia_modules.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        else:
+            res = (jsonify(SimpleStatusCodeResponseModel(
+                        status=404,
+                        message='Error'
+                   )))
+            return make_response(res, 404)
+
+    # @swagger.doc(TODO)
+    def put(self, module_id):
+        """Update an actinia module (process chain templat).
```
Comment (neteler) [inline actinia_gdi/api/actinia_modules.py]:
```suggestion
        """Update an actinia module (process chain template).
```

File: actinia_gdi/api/actinia_modules.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        else:
+            res = (jsonify(SimpleStatusCodeResponseModel(
+                        status=404,
+                        message='Error'
+                   )))
+            return make_response(res, 404)
+
+    # @swagger.doc(TODO)
+    def delete(self, module_id):
+        """Delete an actinia module (process chain templat).
```
Comment (neteler) [inline actinia_gdi/api/actinia_modules.py]:
```suggestion
        """Delete an actinia module (process chain template).
```

**Comment by mmacata** [comment]
Closing this as moved to https://github.com/mundialis/actinia-module-plugin/pull/4

**Pull request closed without merging**